pkg/platform/kube/apis/nuclio.io/v1beta1/zz_generated.deepcopy.go

```diff
@@ -90,7 +150,7 @@ func (in *Project) DeepCopyInto(out *Project) {
 	*out = *in
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
-	out.Spec = in.Spec
+	in.Spec.DeepCopyInto(&out.Spec)
```

Comment:
What's this change?

Reply:
Didn't want to worry about this in the future if project spec became more complex, so I also added a DeepCopyInto for Projects.